eng/pipelines/templates/steps/build-package-artifacts.yml

```diff
@@ -45,6 +45,11 @@ steps:
     displayName: 'Tag scheduled builds'
     condition: and(eq(variables['Build.SourceBranchName'], variables['DefaultBranch']), eq(variables['Build.Reason'],'Schedule'))
 
+  - ${{if eq(variables['System.TeamProject'], 'internal') }}:
+    - template: ../steps/auth-dev-feed.yml
```

Comment:
```suggestion
    - template: ../auth-dev-feed.yml
```

Reply:
[It worked though?](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4121431&view=logs&jobId=1dca5ef1-3811-5712-152d-1052a5815ba2&j=1dca5ef1-3811-5712-152d-1052a5815ba2&t=d4013fff-a382-5e29-31a1-b3bf6a2709de)

Reply:
Ohhhhh k. I see. Yeah I'll definitely make this adjustment.

Reply:
Well  not exactly because to go after the same it's just `auth-dev-feed.yml` or `./auth-dev-feed.yml`. Thanks for the catch regardless!